lib/bolt/util.rb

```diff
@@ -77,6 +77,14 @@ def read_optional_yaml_hash(path, file_name)
         File.exist?(path) ? read_yaml_hash(path, file_name) : {}
       end
 
+      def first_runs_free
+        Bolt::Config.user_path + '.first_runs_free'
```

Comment:
`Bolt::Config.user_path` returns `nil` if there's no homedir, so this will still fail.